tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -20,8 +20,9 @@ var CESIUM_MAP_VIEW = (function() {
         m_view_options,                 // The map view options
         m_terrain_options,              // The map terrain options
         m_image_layer_options,          // The map image layer options
-        m_models_options,               // The map 3D object options
+        m_models,                       // The map 3D object data
         m_entities_options,             // The map entity options
```

Comment:
`m_entities_options` => `m_entities`

Reply:
the m_entities_options should not be changed. It does store the options for entities. not the data like in models and primitives.